src/scaffold/services/api/api.ts

```diff
@@ -0,0 +1,146 @@
+import type { Schema } from "../../schemas/schemas";
```

Comment:
this flattening logic is all subject to change when we start specing out data layer